### PR TITLE
SER-227 As a OAF non client, I want to be able to access CE services through the USSD system so that I can report issues.

### PR DESCRIPTION
**Pull request by Bahatiroben**
https://telerivet.com/p/0c6396c9/service/SV3c5034dacfc21c61/edit

- Dial in the ussd
- enter 99 to change the laguage to english(optional)
- enter 0(zero) to access the non client's menu
- enter 5 to access the issues menu
- choose any issue among the listed
There should be a notice telling you that the issue was submitted and you will be called within 48 hours.

Finishes 
- [SER-278](https://oneacrefund.atlassian.net/browse/SER-278)
- [SER-277](https://oneacrefund.atlassian.net/browse/SER-277)

### CODE REVIEW

**Review by shejasabin — APPROVED** [review]

File: ke-legacy/main.js [hunk 1]
```diff
@@ -3209,13 +3214,21 @@ addInputHandler('CallCenterMenu', function(input) {
         }
         else{
             var create_zd_ticket = require('ext/zd-tr/lib/create-ticket');
-
-            if(create_zd_ticket(client.AccountNumber, sub, contact.phone_number)){
+            var ticketTags = [menu_options[input], 'kenya', 'CallBackUSSD'];
```
Comment (shejasabin) [inline ke-legacy/main.js:3217]: can you add tags to other places these tickets are being created?

Comment (Bahatiroben) [reply ke-legacy/main.js:3218]: @shejasabin have added them

File: ke-legacy/main.js [hunk 2]
```diff
@@ -2938,7 +2941,8 @@ addInputHandler('CallBackPN', function(Input){
     }
     else if (Input =='1'){
         var sub = 'Call back requested for: ' + state.vars.issuetype +' account number : '+ client.AccountNumber+ 'With phonenumber: '+ contact.phone_number;
-        if(create_zd_ticket(client.AccountNumber, sub, contact.phone_number)){
+        var tags = ['rwanda', state.vars.issuetype, 'SerialCode'];
```
Comment (shejasabin) [inline ke-legacy/main.js:2944]: the country is misleading

Comment (Bahatiroben) [reply ke-legacy/main.js]: sorry. was supposed to be Kenya

**Review by nyabongo — APPROVED** [review]

**Review by shejasabin — APPROVED** [review]

**Pull request merged**